scripts/views/mapView.js

```diff
@@ -1,439 +1,391 @@
-          },
-          {
-            'saturation': -93
-          },
-          {
-            'lightness': 31
-          },
-          {
-            'visibility': 'on'
-          }
-        ]
-      },
-      {
-        'featureType': 'road.arterial',
-        'elementType': 'geometry.stroke',
-        'stylers': [
-          {
-            'visibility': 'on'
-          },
-          {
-            'color': '#f3dbc8'
-          },
-          {
-            'saturation': '0'
-          }
-        ]
-      },
-      {
-        'featureType': 'road.arterial',
-        'elementType': 'labels',
-        'stylers': [
-          {
-            'hue': '#bbc0c4'
-          },
-          {
-            'saturation': -93
-          },
-          {
-            'lightness': -2
-          },
-          {
-            'visibility': 'simplified'
-          }
-        ]
-      },
-      {
-        'featureType': 'road.arterial',
-        'elementType': 'labels.text',
-        'stylers': [
-          {
-            'visibility': 'off'
-          }
-        ]
-      },
-      {
-        'featureType': 'road.local',
-        'elementType': 'geometry',
-        'stylers': [
-          {
-            'hue': '#007fff'
-          },
-          {
-            'saturation': -90
-          },
-          {
-            'lightness': -8
-          },
-          {
-            'visibility': 'simplified'
-          }
-        ]
-      },
-      {
-        'featureType': 'road.local',
-        'elementType': 'geometry.fill',
-        'stylers': [
-          {
-            'color': '#ba2317'
-          }
-        ]
-      },
-      {
-        'featureType': 'transit',
-        'elementType': 'all',
-        'stylers': [
-          {
-            'hue': '#e9ebed'
-          },
-          {
-            'saturation': 10
-          },
-          {
-            'lightness': 69
-          },
-          {
-            'visibility': 'on'
-          }
-        ]
-      },
-      {
-        'featureType': 'water',
-        'elementType': 'all',
-        'stylers': [
-          {
-            'hue': '#b6e5fb'
-          },
-          {
-            'saturation': -78
-          },
-          {
-            'lightness': 67
-          },
-          {
-            'visibility': 'simplified'
-          }
-        ]
-      },
-      {
-        'featureType': 'water',
-        'elementType': 'geometry.fill',
-        'stylers': [
-          {
-            'color': '#b6e5fb'
-          },
-          {
-            'saturation': '-72'
-          },
-          {
-            'lightness': '10'
-          }
-        ]
-      },
-      {
-        'featureType': 'water',
-        'elementType': 'geometry.stroke',
-        'stylers': [
-          {
-            'color': '#6496af'
+    mapboxgl.accessToken = accessToken;
+    map = new mapboxgl.Map({
+      container: 'map',
+      style: styleURL,
+      center: continental.center,
+      zoom: continental.zoom,
+      minZoom: 3
```

Comment:
Zooming in is useful in so far as seeing where a townhall will be -- completely agreed.  It just seems like it zooms in way past that point.  As long as all the street names are shown and there's context (around zoom 15) that seems to make more sense?  Else we wind up zooming in so much that we actively lose context.